### DISCUSSION TITLE
Button with input is not working

### DISCUSSION THREAD

**Discussion by sairam220** [primefaces/community#700] [PrimeReact Templates]
<Button  label="Upload Template" className='"p-button-rounded p-button-text mr-2' icon="pi pi-upload" >
                    <input
                        accept=".xlsx"
                        name="file"
                        type="file"
                        value=""
                       hidden
                        onChange={handleFileUploader}
                    />
                </Button>

**Comment by gucal** [comment]
Hi,
Please can you send reproducer
